lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -92,5 +92,193 @@ module Resources
       ]
     }.freeze
 
+    DSTU2_MEDICATION_ADMINISTRATION_MULTIPLE_INGREDIENTS_ENTRY ||= {
```

Comment:
@Mohammad-Isyroqi-Fathan This administration looks odd to me.  I want to look at this in PowerChart before we publish this

Reply:
Yes, this is an example in C1941. However, keep in mind that our code has not been deployed to Prod yet, so the denominator in the contained medications won't be there and the dosage.quantity value returned currently will be different

Reply:
So you will wait for our code before publishing the example, correct?

Reply:
Why isn't the Sodium Chloride ingredient also in the narrative?

Reply:
If I understand the process correctly, I can merge anytime before it's published (so, just merging it will not publish the site). Then, once the code is deployed in Prod, someone will publish fhir.cerner.com, so all merged code here will be published.

Reply:
That someone will need to execute that query again, right, so the response is correct when fhir.cerner.com is published.  FYI, I'm logging a defect on the narrative, but we will continue to release.

Reply:
I modified the query in this PR to match what we are supposed to see after our server code is deployed (added the expected denominator and changed the dosage quantity value. Also updated the dosage quantity value in the narrative)